src/components/Tooltip.tsx

```diff
@@ -194,12 +204,13 @@ interface HelpTooltipProps {
   tooltip: ReactNode
   placement?: Placement
   offset?: number
-  iconSize?: FontAwesomeIconProps['size']
```

Comment:
Opted for simplifying the props, by assuming we'd use a single universal size for the tooltip icon. If we want to deviate, I'd propose we add specific icon sizes in our `theme` and handle that separately.